src/main/java/in/projecteka/consentmanager/user/UserRepository.java

```diff
@@ -13,10 +13,10 @@
 @AllArgsConstructor
 public class UserRepository {
     private final static String INSERT_PATIENT = "Insert into patient(id, " +
-            "first_name, last_name, gender, date_of_birth, phone_number)" +
-            " values($1, $2, $3, $4, $5, $6);";
+            "name, gender, year_of_birth, phone_number)" +
+            " values($1, $2, $3, $4, $5);";
 
-    private final static String SELECT_PATIENT = "select id, first_name, last_name, gender, date_of_birth, phone_number " +
+    private final static String SELECT_PATIENT = "select id, name, gender, year_of_birth, phone_number " +
             "from patient where id = $1";
```

Comment:
Have you written any migration scripts to move first_name or lastname to name in the cm-db-initializer?

Reply:
Yes, will push that once this is approved. Or can raise a PR as well.

Reply:
https://github.com/ProjectEKA/cm-db-initializer/pull/5